### PR TITLE
#4470 [Manager] Create workspace node list card table icon is broken

### PR DESCRIPTION
**Pull request by wu87988622**
### Checklist
- [x] [Review Contribution Guidelines](https://ohara.readthedocs.io/en/latest/contributing.html)
- [x] Add Reviewers (see [Authors](https://github.com/oharastream/ohara#ohara-team))
- [x] Add Assignees (of course, you are the assignee by default!)
- [x] DON'T set a Milestone
- [x] DON'T add any labels (PR labels will be added automatically by Jenkins)
- [x] Link to an issue(You can do so by using the required by keyword: required by #4470)
- [ ] Add unit tests (or please explain why this PR is not worth having new tests)
- [ ] Pass QA (Sometimes you may encounter an existing flaky test which would obstruct you from getting QA passed. If the failed tests are unrelated, you can add a comment in the PR thread)

### CODE REVIEW

**Comment by saivirtue** [comment]
Please retry e2e tests in this PR also :)

**Comment by wu87988622** [comment]
retry manager-e2e

**Comment by saivirtue** [comment]
It looks like we could not click the "other place" except the red circle in node list:
image

Could you help to fix this?

**Comment by wu87988622** [comment]
retry

**Comment by wu87988622** [comment]
retry manager-e2e

**Comment by wu87988622** [comment]
retry manager-e2e

**Comment by wu87988622** [comment]
retry manager-e2e

**Comment by wu87988622** [comment]
retry manager-e2e

**Comment by wu87988622** [comment]
retry manager-e2e

File: ohara-manager/client/src/utils/generate.ts [hunk 1]
```diff
@@ -27,7 +27,7 @@ export const port = ({ min = 5000, max = 65535 } = {}) => {
 };
 
 export const serviceName = ({
-  length = 10,
+  length = 5,
```
Comment (konekoya) [inline ohara-manager/client/src/utils/generate.ts:30]: Is this too long so it's breaking the test?

File: ohara-manager/client/cypress/e2e/topicDialog.test.ts [hunk 2]
```diff
@@ -50,6 +50,9 @@ describe('TopicDialog of Workspace Settings', () => {
       cy.findByText(/^add$/i).click();
     });
 
+    // wait the topics all ready
```
Comment (konekoya) [inline ohara-manager/client/cypress/e2e/topicDialog.test.ts:53]:
```suggestion
    // Wait until all topics are ready
```

Or loaded

File: ohara-manager/client/src/utils/generate.ts (hunk 1 above)
Comment (wu87988622) [reply ohara-manager/client/src/utils/generate.ts:30]: yes

File: ohara-manager/client/cypress/e2e/topicDialog.test.ts (hunk 2 above)
Comment (wu87988622) [reply ohara-manager/client/cypress/e2e/topicDialog.test.ts:53]: thanks

**Comment by wu87988622** [comment]
retry manager-e2e

**Comment by wu87988622** [comment]
retry manager-e2e

**Pull request merged**